### PR TITLE
[Tests-Only] update tests related to deleting public shares

### PR DESCRIPTION
**Pull request by C0rby**
## Description
Update tests related to deleting public links
Maybe the tags of the feature must be changed to make the tests run with the ocis tests. I don't know which should be set.
PR causing the test changes: https://github.com/cs3org/reva/pull/899
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/Sharing.php [hunk 1]
```diff
@@ -2369,7 +2369,11 @@ public function getPublicShareIDByName($user, $path, $name) {
 		$dataResponded = $this->getShares($user, $path);
 		foreach ($dataResponded as $elementResponded) {
 			if ((string) $elementResponded->name[0] === $name) {
-				return (int) $elementResponded->id[0];
+				if (\TestHelpers\OcisHelper::isTestingOnOcis()) {
+					return (string) $elementResponded->id[0];
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/Sharing.php:2373]: @C0rby see PR #37609 - IMO we can always treat this as string. No need for the special `isTestingOnOcis()` check.

I made the separate PR to see if all the tests are happy with it in core. If we merge that PR, then the diffs here will not be needed.

Comment (C0rby) [reply tests/acceptance/features/bootstrap/Sharing.php:2373]: Nice! I wasn't sure if the cast to int was optional so I left it in but yeah after the other PR finishes we can remove this here.

**Comment by phil-davis** [comment]
@C0rby PR #37609 has been merged. You can rebase here and remove the not-needed diffs.

**Review by phil-davis — APPROVED** [review]
PR https://github.com/cs3org/reva/pull/899 was merged into reva last night. And the latest reva was updated in ocis-reva and the latest ocis-reva was updated in ocis. So this fix is implemented everywhere.

**Pull request merged**